vllm/sequence.py

```diff
@@ -497,11 +539,23 @@ def __init__(
         self.computed_block_nums = computed_block_nums
         self.multi_modal_data = multi_modal_data
         self.state = SequenceGroupState() if state is None else state
+        self._token_chunk_size = token_chunk_size
+
+        if self._token_chunk_size is None:
+            if is_prompt:
+                self._token_chunk_size = list(seq_data.values())[0].get_len()
+            else:
+                self._token_chunk_size = 1
```

Comment:
i guess the value here is not meaningful because it's not accessed for decode, now i wonder whether it make sense to use `None`

Reply:
No strong opinion. I just thought it makes more sense as API (that it returns 1 than None). Also less confusing because scheduler returns token_chunk_size == 1 when it is decoding.